ignite/metrics/frequency.py

```diff
@@ -70,10 +70,10 @@ def compute(self):
         # Returns the average processed objects per second across all workers
         return self._n / self._elapsed * time_divisor
 
-    def completed(self, engine, name):
+    def completed(self, engine: Engine, name: str) -> None:
         engine.state.metrics[name] = int(self.compute())
 
-    def attach(self, engine, name, event_name=Events.ITERATION_COMPLETED):
+    def attach(self, engine: Engine, name: str, event_name: Any = Events.ITERATION_COMPLETED) -> None:
```

Comment:
```suggestion
    def attach(self, engine: Engine, name: str, event_name: Events = Events.ITERATION_COMPLETED) -> None:
```

Reply:
this is actually a bit more complicated, because `Frequency` inherits `Metric` the `attach` method should have following typing
```python
def attach(self, engine: Engine, name: str, usage: Union[str, MetricUsage] = EpochWise()) -> None:
``` 
when I change it `Events`, then I have to ignore the typing here.

Reply:
Oh, I see. Seems like `Frequency.attach` signature was not updated when `MetricUsage` was introduced. Thanks for catching !